python/rascaline/calculators.py

```diff
@@ -155,24 +155,36 @@ def compute(
             faster than having to cross the FFI boundary often when accessing
             the neighbor list. Otherwise the Python neighbor list is used.
 
-        :param gradients: List of gradients to compute. If this is ``None`` or
-            an empty list ``[]``, no gradients are computed. The following
-            gradients can be available:
+        :param gradients: List of gradients to compute. If this is ``None``
+            or an empty list ``[]``, no gradients are computed.
+            Gradients are stored inside the different blocks, and can be
+            accessed with ``descriptor.block(...).gradient(<parameter>)``, where
+            ``<parameter>`` is ``"positions"`` or ``"cell"``. The following
+            gradients are available:
 
             - ``"positions"``, for gradients of the representation with respect
               to the atomic positions;
             - ``"cell"``, for gradients of the representation with respect to
-              the cell vectors.
-
-              **WARNING**: these cell gradients are different from the usual
-              virial used to compute the instantaneous stress tensor (the virial
-              is computed as the gradients of the representations with respect
-              to the strain). To recover the typical virial, you'll need to
-              multiply the cell gradients with the cell matrix.
-
-            The gradients are stored inside the different blocks, and can be
-            accessed with ``descriptor.block(...).gradient(<parameter>)``, where
-            ``<parameter>`` is ``"positions"`` or ``"cell"``.
+              the cell vectors. Cell gradients are compute as
+
+              .. math::
+                  \frac{\partial \langle q \vert A \rangle}
+                       {\partial \mathbf{h}}
+
+              where :math:`\mathbf{h}` is the cell matrix and
+              :math:`\langle q \vert A \rangle` is the representation.
+
+              **Note**: The virial is usually computed as the gradient of the
+              representation with respect to the strain :math:`\epsilon`.
+              To recover the typical virial from the cell gradient
```

Comment:
s/virial/expression/

nb: I'm making all these changes because virial indicates the derivative of the energy, so it may be confusing to call the representation gradient "virial"